src/components/expanded-state/chart/chart-data-labels/ChartPriceLabel.js

```diff
@@ -13,13 +17,17 @@ const Label = styled(ChartYLabel)`
   width: 100%;
 `;
 
-export function formatUSD(value, priceSharedValue) {
+export function formatNative(value, priceSharedValue, nativeSelected) {
   'worklet';
   if (!value) {
     return priceSharedValue?.value || '';
   }
   if (value === 'undefined') {
-    return '$0.00';
+    const res =
+      nativeSelected?.alignment === 'left'
```

Comment:
can use `convertAmountToNativeDisplay('0.00', nativeCurrency)` from utilities.js and pass along the nativeCurrency itself (instead of nativeSelected as it is handled for you)

Reply:
Same on line 49

Reply:
also if you use the utility function, then you won't need the `decimals` logic below (on line 32) as this is also handled by the utility function

Reply:
can't do this with reanimated unless we refactor utilities functions